packages/myst-cli/src/transforms/dois.ts

```diff
@@ -182,19 +186,20 @@ export async function getCitation(
   node: GenericNode,
 ): Promise<SingleCitationRenderer | null> {
   if (!doi.validate(doiString)) return null;
-  const data = await resolveDoiOrg(session, doiString, vfile, node);
+  const data = await resolveDoiOrg(session, doiString);
   if (!data) {
     const resolves = await doiOrgResolves(session, doiString);
     const normalizedDoi = doi.normalize(doiString);
     let message: string;
     let note: string | undefined;
     if (resolves) {
-      message = `No citation data available from doi.org for doi:${normalizedDoi}`;
-      note = `To resolve this error, visit ${doi.buildUrl(normalizedDoi)} and add citation info to .bib file`;
+      message = `Citation data from doi.org was not available or malformed for doi:${normalizedDoi}`;
+      note = `To resolve this error, visit ${doi.buildUrl(normalizedDoi)} and add citation info to local BibTeX file`;
```

Comment:
One of the errors that I went through previously was that some DOIs are not "valid" without the `https://doi.org` on them, and so the double validation fails.

```suggestion
      note = `To resolve this error, visit ${doi.buildUrl(doiString)} and add citation info to local BibTeX file`;
```